samples/boards/stm32/power_mgmt/suspend_to_ram/src/main.c

```diff
@@ -13,15 +13,18 @@
 #include <zephyr/pm/device_runtime.h>
 #include <zephyr/drivers/adc.h>
 #include <zephyr/drivers/entropy.h>
+#include <zephyr/drivers/spi.h>
 #include <string.h>
+#include <stdio.h>
+#include <assert.h>
 
 #define SLEEP_TIME_STOP0_MS	800
 #define SLEEP_TIME_STOP1_MS	1500
 #define SLEEP_TIME_STANDBY_MS	3000
 #define SLEEP_TIME_BUSY_MS	2000
 
 static const struct gpio_dt_spec led =
-	GPIO_DT_SPEC_GET(DT_ALIAS(led0), gpios);
+	GPIO_DT_SPEC_GET(DT_ALIAS(led2), gpios);
```

Comment:
Led0 will be more frequently available than led2

Reply:
On Nucleo WBA55CG, led0 is also a pin used by the SPI (I think it's the clock pin), so that's why I changed to led2